packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -26,9 +24,14 @@ contract PluginRepo is
     PermissionManager
 {
     using AddressUpgradeable for address;
-
+    
+    struct Tag {
+        uint8 release;
+        uint16 build;
+    }
+    
     struct Version {
-        uint16[3] semanticVersion;
+        Tag tag;
```

Comment:
Perhaps an explanation comment is needed, or the example comment below that is about to be removed before the merge could be adapted and put here.